lua/auto-session.lua

```diff
@@ -133,11 +133,25 @@ function AutoSession.SaveSession(sessions_dir, auto)
   run_hook_cmds(post_cmds, "post-save")
 end
 
+-- This function checks if the session dir is in ignore list
+local function check_if_in_ignore(sessions_dir)
+  if vim.g.auto_session_ignore and vim.g.auto_session_ignore > 0 then
```

Comment:
Few things here, first you'll probably want to check the number of items in the array-like table.
```suggestion
  if vim.g.auto_session_ignore and #vim.g.auto_session_ignore > 0 then
```

Secondly it would need to support configs from lua as well, so instead of grabbing it straight from `vim.g` you might want to add a config to the `defaultValues` object following the pattern there, and evaluate it from both the VimL and the Lua configs.

Reply:
Alright I'll look into it!